README.md

```diff
@@ -1,2 +1,109 @@
-# bamboo-viability
-🎍 Findings from our EVE Bamboo I viability assessment
+# Bamboo I Viability Assessment
+
+- [Overview](#overview)
+- [Problems areas](#problem-areas)
+  - [Data access](#data-access)
+  - [Design components](#design-components)
+  - [Out-of-band decisions](#out-of-band-decisions)
+  - [Participant accounts](#participant-accounts)
+- [Tasks for CGA](#tasks-for-cga)
+- [Architecture diagram](#architecture-diagram)
+- [Prototype](#prototype)
+
+## Overview
+
+NewAperio has completed a viability assessment for the Bamboo I project (hereafter referred to as _Bamboo_) to ensure that the necessary technology requirements can be met. After a deep dive into the existing [Common Ground for Action (CGA)][cga] application and data model, and collaboration with the [Every Voice Engaged (EVE)][eve] technology team, we’ve uncovered a well-defined problem space and have come up with solutions that will allow the project to move forward.
+
+## Problem areas
+
+These are the problem areas that we identified, and are described in further detail later in the document:
+
+1. Data access
+2. Design components
+3. Out-of-band decisions
+4. Participant accounts
+
+### Data access
+
+#### The problem
+
+Data essential to the Bamboo experience is tied into the CGA environment and not exposed via a web API or other such mechanism. This data includes things like:
+
+- Issue guides
+- Options
+- Actions
+- Drawbacks
+
+#### The solution
+
+NewAperio will build a proxy server application that will connect directly to CGA’s database instance via a read-only connection and expose the required data over a well-defined GraphQL web API.
```

Comment:
I would add this, but this will require EVE's infra to add a Read-Replica to their stack.

Also potentially worth mentioning somewhere that NA will deliver a container that can be deployed into their K8s cluster (so little to no config/devops work necessary on their part).